cve_bin_tool/extractor.py

```diff
@@ -90,7 +90,7 @@ def extract_file_rpm(cls, filename, extraction_path):
             if not inpath("7z"):
                 raise Exception("7z is required to extract rpm files")
             else:
-                subprocess.call(["7z", "x", filename, "-o", extraction_path])
```

Comment:
Here f-string is necessary cause args as a list is not working I have also tried list with f-string but it isn't working.

Reply:
hm, we should probably look into this to see if we can do better, but this should do for now.  Bandit will remind us to code review it again in the future but we could also file a bug about it.  It may be something about the operation on windows.